include/boost/gil/image_processing/numeric.hpp

```diff
@@ -12,6 +12,8 @@
 #include <cmath>
 #include <boost/gil/image_view.hpp>
 #include <boost/gil/typedefs.hpp>
+#include <boost/gil/extension/numeric/kernel.hpp>
+#include <boost/gil/extension/numeric/convolve.hpp>
```

Comment:
It's a good idea to sort includes within blocks, see https://github.com/boostorg/gil/wiki/Include-Directives-Order